python/paddle/fluid/optimizer.py

```diff
@@ -41,7 +41,10 @@ class Optimizer(object):
     but need to use one of it's implementation.
     """
 
-    def __init__(self, learning_rate, regularization=None):
+    def __init__(self,
+                 learning_rate,
+                 regularization=None,
+                 LARS_weight_decay=0.0):
```

Comment:
According to the paper, I think the default value of LARS_weight_decay may be 1.0.